web_widget_kanban_many2one/static/src/js/web_widget_kanban_many2one.js

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2019 Sunflower IT <www.sunflowerweb.nl>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+ */
+
+odoo.define('web_widget_kanban_many2one.kanban_editable', function(require){
+
+    var AbstractField = require('web.AbstractField');
+    var core = require('web.core');
+    var field_registry = require('web.field_registry');
+
+    var _t = core._t;
+
+    var many2one_kanban = AbstractField.extend({
+        template: _t('Kanban_editable_temp'),
+        events: _.extend({}, AbstractField.prototype.events, {
+            'click .selectM2MField': function(event){
+                event.stopPropagation();
+            },
+            'change .selectM2MField': '_onSelectField'
+        }),
+        _render: function () {
+            var el = this.$el;
+            var select = el.find('select');
+            var div = el.closest('.o_kanban_record');
+            var _text = div.find('h2 span').text();
+            var field_name = this.name;
+            var field_data = this.recordData[field_name];
+            var curr_id = field_data ? field_data.data.id : false;
+            // empty item
+            select.append('<option value=""></option>');
+            // options
+            for (var val of this._options) {
+                selected = (val.id == curr_id) ? 'selected' : '';
+                select.append('<option value="' + val.id + '" ' + selected +
+                    '>' + val.name + '</option>');
+            }
+        },
+        _onSelectField: function(){
+            var self = this;
```

Comment:
We only need "self" (or sometimes people call it "that") if we wish to use "this" in a callback function also, for example after '_rpc' returns with success. Otherwise you can just use "this".